@here/harp-mapview/lib/VisibleTileSet.ts

```diff
@@ -598,11 +597,8 @@ export class VisibleTileSet {
             const tiles = renderListEntry.renderedTiles;
             tiles.forEach(tile => {
                 tile.update(renderListEntry.zoomLevel);
-                minElevation = MathUtils.min2(minElevation, tile.minElevation);
-                maxElevation = MathUtils.max2(
-                    maxElevation,
-                    tile.maxElevation + tile.maxGeometryHeight
-                );
+                minElevation = MathUtils.min2(minElevation, tile.geoBox.minAltitude);
+                maxElevation = MathUtils.max2(maxElevation, tile.geoBox.maxAltitude);
```

Comment:
Please check if this is still compatible with datasource / polygons example.

Reply:
It seems to work fine:
https://saeljira.it.here.com/secure/attachment/1217899/harp.gl%20_%20examples.webm